app/services/av_characterizer_service.rb

```diff
@@ -127,6 +132,37 @@ def extract_part(track, part_type)
     }
   end
 
+  def audio_track?(filepath)
```

Comment:
Is it worth adding to the doc at the top of the file that the service also uses ffmpeg now?

Reply:
yes, will add